daemon/cmd/daemon_main.go

```diff
@@ -1588,35 +1593,41 @@ func checkNodePortAndEphemeralPortRanges() error {
 
 func (d *Daemon) launchHubble() {
 	logger := logging.DefaultLogger.WithField(logfields.LogSubsys, "hubble")
-	addresses := option.Config.HubbleListenAddresses
-	if len(addresses) == 0 {
+	if !option.Config.EnableHubble {
```

Comment:
can we move this function to a dedicated file `daemon/cmd/hubble.go`?

Reply:
Absolutely. Will do.